operator/v1/types_dns.go

```diff
@@ -329,6 +345,23 @@ type UpstreamResolvers struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+	// protocolPolicy specifies the protocol to use for upstream DNS
+	// requests. Valid values for protocolPolicy are ForceTCP or omitted.
+	// When omitted, that means no opinion, and the platform uses a default
+	// behavior, which is subject to change over time. Currently, the
+	// default behavior is to use the protocol of the original client
+	// request as the upstream protocol. ForceTCP specifies that the
+	// platform should use TCP for all upstream DNS requests, even if the
+	// client request uses UDP. ForceTCP is useful for UDP-specific issues
+	// such as those created by non-compliant upstream resolvers or UDP
+	// stability problems, but may consume more bandwidth or increase DNS
+	// response time. Note that protocolPolicy only affects the protocol of
```

Comment:
Is it intentional that the godoc in `UpstreamResolvers` has the sentence "ForceTCP is useful for [...]" and the godoc in `ForwardPlugin` does not?